src/components/QueryEditor.tsx

```diff
@@ -207,16 +208,18 @@ export class QueryEditor extends PureComponent<Props, State> {
           </div>
         </div>
         {queryType.value !== QueryType.Builder && schema && (
-          <CodeEditor
-            height={'250px'}
-            language="sql"
-            value={query.rawQuery || ''}
-            onBlur={this.onQueryChange}
-            onSave={this.onQueryChange}
-            showMiniMap={false}
-            showLineNumbers={true}
-            getSuggestions={schema.getSuggestions}
-          />
+          <div aria-label={selectors.components.QueryEditor.CodeEditor.container}>
```

Comment:
likely good if we add a selector in the core component too -- many query editors will just add the CodeEditor

Reply:
good point! will do